addon/sidebar.html

```diff
@@ -3,36 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <title>Sidebar</title>
-    <style>
-      body, html {
-        height: 100%;
-        width: 100%;
-      }
-      #onboarding {
-        margin: 1em;
-      }
-      #browser-container {
-        width: 100%;
-        height: 100%;
-      }
-      iframe {
-        width: 100%;
-        height: 100%;
-        border: none;
-      }
-      #iframe-container {
-        height: 100%;
-        width: 100%;
-        padding-bottom: 20px;
-      }
-      #toolbar {
-        width: 100%;
-        position: absolute;
-        bottom: 0px;
-        height: 20px;
-        border-top: 1px solid #000;
-      }
-    </style>
+    <link rel="stylesheet" href="sidebar.css">
   </head>
   <body>
     <div id="onboarding" style="display: none">
```

Comment:
Do we want to use inline styles, or add some `.hidden { display: none; }` style in addon/sidebar.css?

Reply:
I don't like to add indirection (i.e., class that affects a style instead of changing the style) unless there's a motivation, and right now there isn't one. Maybe if we do animations there would be?